checkstyle/import-control-core.xml

```diff
@@ -142,4 +142,8 @@
     <allow pkg="org.apache.directory" />
     <allow pkg="org.apache.mina.core.service" />
   </subpackage>
+
+  <subpackage name="clients">
+    <allow pkg="org.apache.kafka.clients" />
+  </subpackage>
```

Comment:
Is this necessary? We have other, existing integration tests that reference the `clients` sub-package and don't complain. Is this a new requirement?

Reply:
Could you point out which existent integration tests don't complain? I think `kafka.clients.consumer` is a new package. Thank you.